packages/nodes-fs/vite.config.mts

```diff
@@ -4,6 +4,12 @@ import { defineConfig } from 'vitest/config';
 export default defineConfig({
 	test: {
 		include: ['tests/**/*.test.[jt]s?(x)'],
-		globals: true
+		globals: true,
+		server: {
+			deps: {
+				// to ensure importing CSS files is working
+				inline: ['@tokens-studio/ui']
```

Comment:
Do we need to update this for the other packages?

Reply:
Yeah if we're using Vitest and using our latest DS that uses CSS modules imports